src/read/macros.rs

```diff
@@ -135,69 +390,140 @@ pub enum DebugMacInfoItem<R> {
 
 /// Iterator over the entries in the `.debug_macinfo` section.
 #[derive(Clone, Debug)]
-pub struct DebugMacInfoIterator<R: Reader> {
+pub struct DebugMacroIterator<R: Reader> {
```

Comment:
```suggestion
pub struct MacroIter<R: Reader> {
```

This is more consistent with other naming in gimli.